### PR TITLE
Add fk_nominal_diameter for qwat_od.valve [feature]

### LINKED ISSUE #305: Valve nominal diameter
I'm facing an issue on  `diameter_nominal` from  `qwat_od.valve` 

This column is a varchar(10) type without foreign key. For me, it's ambiguous and the demonstration data confirm my doubt about that [1]

With this liberty of input, you can enter: DN, NPS, inside/outside size, etc.

For me, it should be only the size from the ISO 6708 (DN 50, DN 100, etc). So an integer type (millimeter value).

What is your practice and convention? @ponceta @kandre @tudorbarascu 

[1]
```
SELECT DISTINCT diameter_nominal
FROM qwat_od.valve
```

returns

```
90/63
80
1 1/4''
125
2500
200/63
160-63
50
1 1/4"
160/63
63
250
1/2''
200
65
32
1"
70
1.5"
120
200/2"
3/4"
60
40
90
1,25"
1,25
150
1 1/2"
300
180
1,5"
100
250/2"
2"
125/63
2''
1.25"
63/63
```

### PR DESCRIPTION
**Pull request by lbartoletti**
This PR adds a new value lists for nominal diameter and a foreign key for qwat_od.valve.

Fix #305

### CODE REVIEW

**Comment by haubourg** [comment]
Looks good. 

As this is a feature, can you add the tag [feature] to the merge commit and a minimal description of it so that we don't have to navigate to the issue to build the changelog? Thanks!

**Comment by lbartoletti** [comment]
@tudorbarascu @kandre @ponceta 

Please, can you test and improve this [part](https://github.com/qwat/qwat-data-model/pull/312/files#diff-e33520dab03bd3e96c4377d56560327dR4-R44) with your values?

**Comment by tudorbarascu** [comment]
@lbartoletti Will check and get back at you in a few days.

File: update/delta/delta_1.3.4_004_create_fk_nominal_diameter.sql [hunk 1]
```diff
@@ -0,0 +1,106 @@
+ALTER TABLE qwat_od.valve ADD column fk_nominal_diameter integer;
+
+UPDATE qwat_od.valve
+SET fk_nominal_diameter = (
+CASE
+WHEN diameter_nominal = '90/63' THEN 7008
```
[the remark this answers is not in the document]
Comment (lbartoletti) [reply update/delta/delta_1.3.4_004_create_fk_nominal_diameter.sql]: I added `WHEN diameter_nominal = '1/2"' THEN 7002` others are already present. Thanks @tudorbarascu

**Pull request merged**